testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/Hibernate2LCacheStatsTestCase.java

```diff
@@ -91,8 +90,6 @@ public class Hibernate2LCacheStatsTestCase {
     public static void beforeClass() throws NamingException {
         // TODO This can be re-looked at once HHH-13188 is resolved. This may require further changes in Hibernate.
         AssumeTestGroupUtil.assumeSecurityManagerDisabled();
-        // This test needs to be recompiled against Hibernate ORM 6 (WFLY-16178) in order to pass.
```

Comment:
I passed  `
cd testsuite/integration/basic
mvn clean test -Dtest=HibernateNativeAPINaturalIdTestCase -Dsecurity.manager`

Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
```

With this change:
git diff
diff --git a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/naturalid/HibernateNativeAPINaturalIdTestCase.java b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/naturalid/HibernateNativeAPINaturalIdTestCase.java
index 00b078d3f1..565e7c0efe 100644
--- a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/naturalid/HibernateNativeAPINaturalIdTestCase.java
+++ b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/naturalid/HibernateNativeAPINaturalIdTestCase.java
@@ -23,7 +23,7 @@
 package org.jboss.as.test.integration.hibernate.naturalid;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assume.assumeTrue;
+// import static org.junit.Assume.assumeTrue;
 
 import jakarta.ejb.EJB;
 import javax.naming.InitialContext;
@@ -32,7 +32,7 @@ import javax.naming.NamingException;
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.arquillian.test.api.ArquillianResource;
-import org.jboss.as.test.shared.util.AssumeTestGroupUtil;
+// import org.jboss.as.test.shared.util.AssumeTestGroupUtil;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.StringAsset;
@@ -93,9 +93,9 @@ public class HibernateNativeAPINaturalIdTestCase {
     @BeforeClass
     public static void beforeClass() throws NamingException {
         // TODO This can be re-looked at once HHH-13188 is resolved. This may require further changes in Hibernate.
-        AssumeTestGroupUtil.assumeSecurityManagerDisabled();
+        // AssumeTestGroupUtil.assumeSecurityManagerDisabled();
         // This test needs to be recompiled against Hibernate ORM 6 (WFLY-16178) in order to pass.
-        assumeTrue(System.getProperty("ts.ee9") == null && System.getProperty("ts.bootable.ee9") == null);
+        // assumeTrue(System.getProperty("ts.ee9") == null && System.getProperty("ts.bootable.ee9") == null);
 
         iniCtx = new InitialContext();
     }
```

So we likely can clean up the tests that mention WFLY-16178.  Should we do that separately from this pr?

Reply:
@scottmarlow Your change ^^^ looks like it is aimed at the comment about HHH-13188. My impression is that is different from the WFLY-16178 comment.

A change like that should be a different PR, as this one is just about ts.ee9/ts.bootable.ee9 and similar things.

Reply:
Note that instead of removing these two lines I could have changed L95 to

assumeFalse(AssumeTestGroupUtil.isWildFlyPreview())

But that seemed silly since I knew the test passing against EE 10 std WF meant it should pass against WF Preview.